apps/graphql/src/mocks/itinerary.js

```diff
@@ -0,0 +1,108 @@
+// @flow
+
+const getCountry = () => ({
+  id: 'aaa',
+  name: 'Norway',
+  code: 'NO',
+  slug: '',
+  flagURL: '',
+});
+
+const vehicle = {
+  type: 'bus', // bus | train | aircraft,
+  uniqueNo: '1234',
+};
+
+const provider = {
+  id: 'ads',
+  name: 'Ryan Air', // RegioJet, Rayner, Emirates, etc….
+};
+
+const getLocation = (locationId, name) => ({
+  id: 'aa',
+  locationId: locationId,
+  name: name,
+  timezone: 'UTC+1',
+  country: getCountry(),
+});
+
+const price = {
+  currency: 'EUR',
+  amount: 1234,
+};
+
+const getDate = (...date) => ({
+  local: new Date(...date),
+  utc: new Date(Date.UTC(...date)),
+});
+
+const getSegment = (origin, destination, departureTime, arrivalTime) => ({
+  arrivalTime: getDate(2019, 3, 14, 10, 5),
+  departureTime: getDate(2019, 3, 11, 10, 5),
+  destination: destination,
+  duration: 100,
+  id: 'asdfg',
+  origin: origin,
+  provider: provider,
+  vehicle: vehicle,
+  // "connections": Segment[]  // the array of related Segments
```

Comment:
I'm not sure if we will need this.

Reply:
I don't know probably we can avoid this for now, but I think will be useful know if the current segment is part of a connection. So maby ` "connections"?: Array<SegmentID>` will be enough for this information.